iocore/net/P_NetVConnection.h

```diff
@@ -81,3 +85,21 @@ NetVConnection::get_local_port()
 {
   return ats_ip_port_host_order(this->get_local_addr());
 }
+
+TS_INLINE sockaddr const *
+NetVConnection::get_proxy_protocol_addr(ProxyProtocolData_t src_or_dst)
+{
+  if (src_or_dst == PROXY_PROTO_SRC) {
```

Comment:
Like, here you are treating this really like a boolean, i.e. you don't check any other possible values of src_or_dst).